app/controllers/issues/evidence_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Issues::EvidenceController < IssuesController
+  before_action :set_issues, only: [:index]
+  before_action :set_affected_nodes, only: [:index]
+
+  def index
+    @evidence_columns = ['Node'] | all_evidence_columns | ['Created by', 'Created',  'Updated']
+
+    render layout: false
+  end
+
+  private
+
+  def all_evidence_columns
+    @issue.evidence
+          .map { |evidence| evidence.fields.keys }
+          .flatten
+          .uniq - ['Title', 'Label']
+  end
+
+  def set_issues
+    @issues = Issue.where(node_id: current_project.issue_library.id)
```

Comment:
Why do you need to load this collection?

Reply:
good question, removed.